src/main/java/hudson/plugins/sshslaves/verifiers/ManuallyProvidedKeyVerificationStrategy.java

```diff
@@ -31,6 +31,8 @@
 
```

Comment:
We will probably want to introduce a new class in this file called

public boolean isClass(String className) {
    try  {
        Class.forName(className);
        return true;
    }  catch (ClassNotFoundException e) {
        return false;
    }
}

then on line 97

we will want to do  } else if (isClass("ED25519KeyAlgorithm") && "ssh-ed25519".equals(algorithm)) {

and same for the rest relating the class name with the other implementations.

This is so we can keep existing backwards compatible with java 6 and jenkins 1.x.

http://stackoverflow.com/questions/5794227/how-to-check-if-class-exists-somewhere-in-package